src/test/java/seedu/eduke8/stats/StatsCalculatorTest.java

```diff
@@ -105,7 +94,7 @@ void calculateProgressionPercentage_userProgressExceedTotalAvailableProgress_exp
 
     @Test
     void calculateTotalQuestionCount_twoTopicsWithThreeQuestionsEach_returnsCorrectCount() {
-        StatsCalculator statsCalculator = new StatsCalculator(createTestTopicList().getInnerList());
+        StatsCalculator twoTopicsStatsCalculator = new StatsCalculator(createTestTopicList().getInnerList());
```

Comment:
Thanks for making this clear that  it uses a different construction from the rest!